crates/zizmor/src/audit/known_vulnerable_actions.rs

```diff
@@ -110,33 +111,77 @@ impl KnownVulnerableActions {
                 _ => Severity::Unknown,
             };
 
-            results.push((severity, vuln.ghsa_id));
+            // Get the first patched version from the first vulnerability in the advisory
+            let first_patched_version = vuln
+                .vulnerabilities
+                .first()
+                .and_then(|v| v.first_patched_version.clone());
```

Comment:
Noting: this looks great to me for now, but we _might_ want to additionally filter by semver compatibility between versions. But that can be a future problem 🙂